swagger.yaml

```diff
@@ -259,6 +259,12 @@ paths:
         required: false
         schema:
           type: string
+      - name: status
+        in: query
+        description: Status of device being '0' for off or '1' for on
+        required: false
+        schema:
+          type: string
```

Comment:
Kinda thinking this should be number even though its a string in the query string, its parsed as a number.